src/main/java/com/altenheim/kalender/controller/viewController/SettingsViewController.java

```diff
@@ -36,20 +38,26 @@ public void changeContentPosition()
     }
 
     @FXML
-    void buttonClicked(ActionEvent event) throws IOException
-    {
+    void buttonClicked(ActionEvent event) throws IOException, InterruptedException {
         var button = (Button)event.getSource();
         if(button.equals(btnExport))
-            exportController.exportCalendarAsFile(allCalendars.getSpecificCalendarByIndex(0),
-                    settings.getPathToIcsExportedFile());
+        {
```

Comment:
Gehört hier natürlich nicht rein, hatte nach dem Sprint die noch leere Settingsview missbraucht um über die Buttons testweise Funktionen aufzurufen. Hier jetzt Test der API bei Klick auf den Exportbutton. Kommt aber natürlich alles noch raus, ebenso wie die ganzen unnötg injizierten Objekte in dieser Klasse (war auch nur zum Testen).